modules/installation-requirements-user-infra.adoc

```diff
@@ -93,7 +93,7 @@ Each cluster machine must meet the following minimum requirements:
 |120 GB
 
 |Compute
-|{op-system} or RHEL 7.6
+|{op-system}
```

Comment:
The original line is true for bare metal, so please add a conditional for the Z case.

Reply:
Fixed.